### PR TITLE
[Refactor] spaces → tabs

### PR DESCRIPTION
**Pull request by falsandtru**
Can we do 1 or 2 optimizations?

### CODE REVIEW

File: implementation.js [hunk 1]
```diff
@@ -161,7 +159,7 @@ module.exports = function from(arrayLike) {
 		items = arrayFromIterable;
 		len = arrayFromIterable.length;
 	} else if (isString(items)) {
-		return strMatch.call(items, /[\uD800-\uDBFF][\uDC00-\uDFFF]?|[^\uD800-\uDFFF]|./g) || [];
+		return ''.match.call(items, /[\uD800-\uDBFF][\uDC00-\uDFFF]?|[^\uD800-\uDFFF]|./g) || [];
```
Comment (JakeChampion) [inline implementation.js]: Here you are creating a new empty string, traversing the prototype for `match` and after executing it you are throwing away the empty string, I think the previous approach of extracting and caching the `match` method is the better option to take.

Comment (falsandtru) [inline implementation.js]: I'll drop this change if we don't need it.

Comment (ljharb) [inline implementation.js]: Yes, the previous approach is far superior.

Comment (falsandtru) [inline implementation.js]: Okay, fixed.

**Comment by ljharb** [comment]
While we're at it, can we make line 11 in `index.js` use tabs also?

**Comment by falsandtru** [comment]
Sure.

**Pull request merged**